st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -343,7 +352,7 @@ def _run_action(self, action_node, parent_execution_id, params, wait_for_complet
             LOG.debug('%s: Task notify set to: %s', action_node.name, liveaction.notify)
 
         liveaction.context = {
-            'parent': str(parent_execution_id),
+            'parent': parent_context,
```

Comment:
What's our migration path for this change?

If I ignore the read path (listing, etc.), I think it only affects the actions which get started (requested / scheduled) with an old version and executed by a new version. Is this correct?